### PR TITLE
Support non-image attaches

### LINKED ISSUE #651: Support non-image attaches
Create a link tag instead of img just like github does

### PR DESCRIPTION
**Pull request by DenisVorop**
## PR includes

- [x] Feature

## Related issues

Resolve #651 

### .PNG

https://github.com/taskany-inc/bricks/assets/95316053/7bd666ec-075b-4a02-8e55-d4341230fec5

### .JPEG

https://github.com/taskany-inc/bricks/assets/95316053/355906ef-b3dd-401e-8b12-2f7d251247ee

### .PDF

https://github.com/taskany-inc/bricks/assets/95316053/11694df0-5a59-475a-af28-d234f0f8920f

### CODE REVIEW

File: src/components/FormEditor.tsx [hunk 1]
```diff
@@ -251,7 +251,14 @@ const StyledUploadIcon = styled(IconAttachOutline)`
     padding-right: ${gapXs};
 `;
 
-const mdImageLink = (url: string) => `![](${url})`;
+const mdImageLink = (file: { type: string; filePath: string; name: string }) => {
```
Comment (LamaEats) [inline src/components/FormEditor.tsx:254]: Maybe it's worth getting such a function from props?

File: src/components/FormEditor.tsx [hunk 2]
```diff
@@ -378,7 +385,7 @@ export const FormEditor = React.forwardRef<HTMLDivElement, FormEditorProps>(
                         endLineNumber: p.lineNumber,
                         endColumn: p.column,
                     },
-                    text: files.map((url) => mdImageLink(url)).join(', '),
+                    text: files.map((file) => mdImageLink(file)).join(', '),
```
Comment (LamaEats) [inline src/components/FormEditor.tsx:388]: Really needs join values with comma in this case? Probably for images it must be a `new line` character

File: src/components/FormEditor.tsx [hunk 3]
```diff
@@ -251,7 +258,18 @@ const StyledUploadIcon = styled(IconAttachOutline)`
     padding-right: ${gapXs};
 `;
 
-const mdImageLink = (url: string) => `![](${url})`;
+const mdImageLink = (file: File) => {
```
Comment (LamaEats) [inline src/components/FormEditor.tsx:261]: rename this function please, now its not only for images :) ex `fileToMD`

**Review by LamaEats — APPROVED** [review]

**Review by asabotovich — APPROVED** [review]

**Pull request merged**